src/js/utils/ballotSearchPriority.jsx

```diff
@@ -22,15 +22,15 @@ export default function ballotSearchPriority (originalString, item, ignoreDescri
   for (let i = 0; i < wordsArray.length; i++) {
     searchNeedleString = wordsArray[i].toString();
     oneWordScore = 0;
-    if (countMatches(searchNeedleString, item.ballot_item_display_name)) {
+    if ((item.ballot_item_display_name && item.ballot_item_display_name.includes(searchNeedleString))) {
```

Comment:
Hi @SailingSteve these changes were in a recent pull request of min and need to all stay with the countMatches function (as opposed to rolling them back to ".includes".

Reply:
This was unintentional, I have reversed the three of them in my next PR.  (Hopefully today or tomorrow)..